### DISCUSSION TITLE
Hardware for container?

### DISCUSSION THREAD

**Discussion by Thormus** [mrlt8/docker-wyze-bridge#122] [Q&A] [answered]
Does it matter what hardware runs this container? Do I need to pass quick sync through the container? Would an rpi4 work?

**Comment by mrlt8** [comment, marked as answer]
Should work fine on the pi 4. Unless you're resizing or doing some processing on the video, the container should just copy the raw h264 from the tutk stream to rtsp.

**Reply by Thormus** [reply]
getting 3-5 fps in blue iris using 1 rpi4 and 2 wyze v3 cameras. is it possible to utilize quicksync?

**Reply by mrlt8** [reply]
I believe quicksync only works for decoding and encoding? As you can see, we only "copy" the raw h264 frames from the camera using the tutk library and send them to the rtsp stream without any decoding/encoding: https://github.com/mrlt8/docker-wyze-bridge/blob/b6f6f77f730ee59f5ede0c9b7a984f4f59c02ff5/app/wyze_bridge.py#L339-L340

Could you use QuickSync in blue iris when decoding the stream?

**Reply by Thormus** [reply]
Gotcha. Yeah my blue iris machine is utilizing quick sync. I might try running the container on something more powerful to see if I can get more frames.

**Reply by mrlt8** [reply]
If that doesn't improve things, then it could be a network issue as I was able to get at least 5 cams to stream without any issue on a Pi 4.

**Reply by Thormus** [reply]
Honestly the streams look good and clear and are very stable. Just seeing if I can optimize for more frames. Even deep stack is working well.
